cms/envs/common.py

```diff
@@ -1142,3 +1142,4 @@
 
 # OpenID Connect issuer ID. Normally the URL of the authentication endpoint.
 OAUTH_OIDC_ISSUER = 'https://www.example.com/oauth2'
+OAUTH_ID_TOKEN_EXPIRATION = 300
```

Comment:
Nitpick: When using a setting measured in seconds to configure a certain number of minutes, I think it's more readable to use `5 * 60`.